index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="hero">
+            <nav class="navbar">
+                <h2 class="logo">Cobby Elson</h2>
+                <button type="button" class="hamburger-btn">
+                    <span class="toggler-icon top-bar"></span>
+                    <span class="toggler-icon middle-bar"></span>
+                    <span class="toggler-icon bottom-bar"></span>
+                </button>
+            </nav>
+            <div class="about-section">
+                <div class="container-for-about-section">
+                    <div class="about-section-item">
+                        <p class="intro  ">I’m Cobby. Glad to see you!</p>
+                    </div>
+                    <div class="about-section-item">
+                        <p class="profile">I’m a software developer! I can help you build a product , 
+                            feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, 
+                            don’t hesitate to contact me.
+                        </p>
+                    </div>
+                    <div class="about-section-item">
+                        <a href="http://" class="lets-connect">LET’S CONNECT</a>
+                        <div class="social-media-icons">
+                            <img src="./assets//twitter.png" alt="twitter">
+                            <img src="./assets//Linkedin icon.png" alt="linkedin">
+                            <img src="./assets/socialIcon.png" alt="social">
+                            <img src="./assets/git.png" alt="git">
+                            <img src="./assets/twohandicon.png" alt="my pub">
+                        </div> 
+
+                    </div>
+                                          
+                </div>
+            </div>
+        </div>
+    </header>
```

Comment:
You did well with the toolbar section, however, this `header` section should end on line 24, please the close `header` should be on line 24 😄

Reply:
Alright I understand ,so after putting the header element on line 24. 
- Should i also put the left over code into a section tag ?
- Must there be a wrapper for the whole  header and the next section (like wrapping both components with a div)